gradle.properties

```diff
@@ -1,3 +1,2 @@
 package=co.com.bancolombia
 systemProp.version=1.8.3
```

Comment:
Please increment the version of the plugin, in readme.md, gradle.properties and Constants.java